apps/api/src/app/widgets/e2e/get-unseen-count.e2e.ts

```diff
@@ -68,7 +67,7 @@ describe('Unseen Count - GET /widget/notifications/unseen', function () {
 
     const messages = await messageRepository.findBySubscriberChannel(
       session.environment._id,
-      subscriberProfile!._id,
+      String(subscriberProfile?._id),
```

Comment:
Each of these statements have the same result on typings and functionality, but the underlying issue still exists - what if `subscriberProfile` is nullish? `String(undefined)` just results in `'undefined'` as a string.

Is this fixing something in this PR?

Reply:
The goal of these changes is to reduce the TS warnings shown that just provide noise to the linting.
The changes casting to String is to get rid of few TypeScript warnings due the nature of the implementation of the values retrieved for the subscriber. It happens in more other test files. Technical debt we have, indeed.

The rule warning is that we are not allowing non-null assertions in the ruleset. Considering these changes were done inside of tests that if we don't have a subscriber it must fail or even if the id used to retrieve results from the database is `'null'` or `'undefined'` the tests must fail. I found that was the less intrusive change for this situation without having to touch tenths of files more.